pkg/deploy/controller/imagechange/controller.go

```diff
@@ -111,10 +111,10 @@ func (c *ImageChangeController) Handle(stream *imageapi.ImageStream) error {
 	}
 
 	if anyFailed {
-		return fatalError(fmt.Sprintf("couldn't update some deployment configs for trigger on image stream %q", imageapi.LabelForStream(stream)))
```

Comment:
I don't think it should ever have been.  Other errors probably should be,
but not conflicts.  I can add that guard here.

On Sun, Jul 10, 2016 at 10:54 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/deploy/controller/imagechange/controller.go
> <https://github.com/openshift/origin/pull/9578#discussion_r70196736>:
>
> > @@ -111,10 +111,10 @@ func (c *ImageChangeController) Handle(stream *imageapi.ImageStream) error {
> >  	}
> >
> >  	if anyFailed {
> > -		return fatalError(fmt.Sprintf("couldn't update some deployment configs for trigger on image stream %q", imageapi.LabelForStream(stream)))
>
> if that's what we want, this looks fine. I wasn't sure why this was
> originally considered a fatal error
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/a850e5e780643ff932482d6a9224cbfbb137f332..ddd80e057bc3163119b64497ec1f0e1ba8b532fb#r70196736>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p1rFV8PQtF8WuseT5FWeKxE4fr9aks5qUbBjgaJpZM4I-xTE>
> .
>